spec/lib/dispatch-rider/subscriber_spec.rb

```diff
@@ -107,12 +117,13 @@ def process(options)
       context "when process is interupted" do
         before do
           stub_const("Interupter", Class.new(DispatchRider::Handlers::Base) {
-            def process(options)
-              Process.kill("INT", 0)
-            end
-          })
+                                   def process(options)
+                                     Process.kill("INT", 0)
+                                   end
+                                 })
```

Comment:
`end` at 123, 33 is not aligned with `Class.new(DispatchRider::Handlers::Base) {` at 119, 35 or `stub_const("Interupter", Class.new(DispatchRider::Handlers::Base) {` at 119, 10